Css/reset.css

```diff
@@ -0,0 +1,9 @@
+body,
+body * {
+margin: 0;
+padding: 0;
+vertical-align: top;
+box-sizing: border-box;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
We should choose one of the fonts: Gill Sans or Poppins.
Right now, reset.css and main.css conflicts each other.